app-frontend/src/app/pages/editor/bucket/bucketedit.html

```diff
@@ -0,0 +1,92 @@
+<div class="container column-stretch">
+  <!-- Sidebar (container is in parent view) -->
+  <div class="sidebar scene-list-pane" ng-if="$ctrl.showLayerList" >
+    <!-- scene listing -->
+    <div class="sidebar-static">
+      <h5 class="sidebar-title">
+        Selected scenes: {{ $ctrl.selectedScenes.size }}
+      </h5>
+      <div class="sidebar-actions">
+        <select>
+          <option selected>Action</option>
+          <option><i class="icon-load"></i> Sync histogram</option>
+          <option><i class="icon-ccw"></i> Reset changes</option>
+          <option><i class="icon-trash"></i> Delete</option>
+        </select>
+        <button class="btn btn-square"
+                ng-class="{'btn-secondary': !$ctrl.shouldSelectAll()}"
+                ng-click="$ctrl.onToggleSelection()">
+          <i ng-class="{'icon-plus': $ctrl.shouldSelectAll(),
+                        'icon-cross': !$ctrl.shouldSelectAll()}"></i>
+        </button>
+      </div>
+    </div>
+    <div class="sidebar-scrollable" id="infscroll">
```

Comment:
Not sure we want an infinite scroll - think it might make sense to retrieve all of the ids up front, so when we do a select all, we can actually include all the ids in any requests we make to the server for stuff like color correction.